modules/upgrade/pages/server-minor-upgrade-uyuni.adoc

```diff
@@ -0,0 +1,9 @@
+[server-minor-upgrade]
+= Server - Minor Upgrade
+
+Several times the year, the {productname} team releases minor upgrades of the {productname} Server.
+These updates offer improved or completely new features.
+For more information, see the Release Notes.
```

Comment:
I would do this a big warning with flashing lights, sound and dancing geckos. Reading the release notes is *critical* as sometimes there are manual steps required, or at least things to consider, even when you are not changing the base OS.

Reply:
ok, I added some such warning.